### ISSUE TITLE
how to use it

### ISSUE THREAD

**Issue by sailseem** [gnn4dr/DRKG#27]
can you show some examples by using this model to predict some drugs on certain diseases？what's the input？ thanks a lot

**Comment by mufeili** [comment]
Have you checked the "Notebooks for analyzing DRKG" in README?

**Comment by sailseem** [comment]
Yes, well, what i meant is that the input. If I understand this right, so I can input any disease symptom, genes, pathways, biological pathways as long as it exists in the data-source of DRKG. Besides, I should input the drug candidate lists, and the output will be the score, which is minus, more close to 0, will be the better choice, am i right? Thank you very much

**Comment by mufeili** [comment]
> Yes, well, what i meant is that the input. If I understand this right, so I can input any disease symptom, genes, pathways, biological pathways as long as it exists in the data-source of DRKG. Besides, I should input the drug candidate lists, and the output will be the score, which is minus, more close to 0, will be the better choice, am i right? Thank you very much

@classicsong Can you take a look?
